src/Sylius/Bundle/ApiBundle/Serializer/CustomerDenormalizer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Serializer;
+
+use DateTimeInterface;
```

Comment:
As per symfony CS, classes like `\DateTimeInterface` shouldn't be explicitly `use`d
![image](https://github.com/Sylius/Sylius/assets/870747/2b5187d1-2d95-4900-86e7-244395bf36d1)

https://symfony.com/doc/current/contributing/code/standards.html#structure